src/js/newsletter/components/SubscribeForm.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useCallback } from "react"
+import EmailInput from "./EmailInput"
+import SubscribeButton from "./SubscribeButton"
+import ConditionsCheckbox from "./ConditionsCheckbox"
+
+function SubscribeForm({ onSubscriptionSend }) {
+  const [conditionsAcknowledged, setConditionsAcknowledged] = useState(false)
+
+  const toggleConditionsAcknowledged = useCallback(() => {
+    setConditionsAcknowledged(!conditionsAcknowledged)
+  }, [conditionsAcknowledged, setConditionsAcknowledged])
+
+  const handleSubmit = useCallback(
+    event => {
+      event.target.submit()
```

Comment:
mmh, that confuses me, shouldn that be called automatically by the browser unless you do `preventDefault`.

Reply:
Understandable. I was tinkering around that problem for a while. The problem is that the closing will remove the form at the DOM. This lead to a not succeeding form action, since it was not able to get the field values. Therefore I did the submit manual.
I did a new commit with a solution which could be better since it reduces the complexity of the form  component. It does this by letting the banner hide itself and not remove it from the DOM. Please tell me if such solutions is better code. Thanks!

Reply:
Ah, makes sense. I would say removing something from the DOM while it is doing something can lead to problems, so I would say the current solution is the preferred.